pawn/scripting/include/ripext/http.inc

```diff
@@ -1,3 +1,5 @@
+#define MAX_PATH_LENGTH 256
```

Comment:
* The name should be prefixed with `SM_RIPEXT_` to avoid conflicts.
* The maximum URL length accepted by most web servers seems to be 8K.

Reply:
https://httpd.apache.org/docs/current/mod/core.html#limitrequestline
https://httpd.apache.org/docs/current/mod/core.html#limitrequestfieldsize